engine_test.go

```diff
@@ -331,6 +323,33 @@ func TestQueries(t *testing.T) {
 		}
 	})
 }
+func TestDescribe(t *testing.T) {
```

Comment:
in parallel it has different output, instead of adding two cases for all other queries, i preferred to extract this one to its own test